### PR TITLE
Remove XML parsing from ProfilePolicy

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

File: docs/changes/v11.1.0/API-Changes.adoc [hunk 1]
```diff
@@ -8,3 +8,6 @@ Use `org.mozilla.jss.symkey.SessionKey` from JSS instead.
 == com.netscape.cmscore.request.CertRequestConstants has been removed ==
 The `com.netscape.cmscore.request.CertRequestConstants` class has been removed.
 Use `com.netscape.certsrv.request.IRequest` constants instead.
+
+== CACertRequest files loaded via the CLI must now be JSON format ==
+The use of `XML` is deprecated since `v9.0.0` and in many cases the XML support has been dropped already.
```
Comment (edewata) [inline docs/changes/v11.1.0/API-Changes.adoc:13]: Hmm.. shouldn't it be `v11.0` when we started changing XML to JSON?

**Review by edewata — APPROVED** [review]
I have one question, but overall it LGTM. Just a note, that QE might be required to update their tests because of this change.

**Comment by ckelleyRH** [comment]
> I have one question, but overall it LGTM. Just a note, that QE might be required to update their tests because of this change.

I think they will. This change came about as a result of helping @cpinjani diagnose an issue he is seeing (and also one of the bugs I am working on) so I imagine we will hit on something.

**Comment by ckelleyRH** [comment]
Thanks @edewata :+1:

**Pull request merged**